search-api/src/search_api/resources/v1/purchase_requests.py

```diff
@@ -35,6 +35,11 @@ def get(request_id=None):
         if not account_id:
             return resource_utils.account_required_response()
 
+        # updates user information with given jwt
+        user = User.get_or_create_user_by_jwt(g.jwt_oidc_token_info)
```

Comment:
NOTE: I added this in so that it would update the user info in our db when they attempt to get their active requests, but it is not used for anything